paddle/fluid/operators/multinomial_op.cc

```diff
@@ -53,12 +53,18 @@ class MultinomialOp : public framework::OperatorWithKernel {
 
     auto x_dim = ctx->GetInputDim("X");
     int64_t x_rank = x_dim.size();
+    PADDLE_ENFORCE_EQ(
+        x_rank > 0 && x_rank <= 2, true,
```

Comment:
use PADDLE_ENFORCE_GT and PADDLE_ENFORCE_LE instead.

Reply:
done